csharp/ql/test/library-tests/ir/rangeanalysis/test.cs

```diff
@@ -0,0 +1,78 @@
+class RangeAnalysis {
+  static void sink(int val) {
```

Comment:
Capital letter + `{` on next line (also throughout this file).

Reply:
Fixed